### ISSUE TITLE
Is comparison with other methods on LineMod fair?

### ISSUE THREAD

**Issue by imankgoyal** [ethnhe/PVN3D#43]
Hi, 
I really like the paper and thanks for open-sourcing the code. As explained in #3, a different model is trained for each object category on LineMod. However, I am confused whether the comparison with other methods (specifically DenseFusion and DeepIM) on LineMod is fair. This is because, other methods seem to use a single model during test time. Hence, object category needs to be inferred by the model at test time. I would very much appreciate if you can please correct my understanding.
Thanks!

**Comment by ethnhe** [comment]
The setting on LineMod is following previous work like PVNet. In fact, Densefusion used pre-trained semantic segmentation models to get the target object region and category labels in both LineMod and YCB datasets, which is not inferred by the DenseFusion models.  While on the YCB dataset, our single model gets instance semantic segmentation and object poses of the whole scene.

**Comment by ethnhe** [comment]
In fact, the segmentation result, especially the category label [DenseFusion](https://github.com/j96w/DenseFusion) used on the LineMod dataset is good enough. You can also try to replace the category label to ground-truth label and inference DenseFusion to see the result, the difference is small. I mentioned YCB because I think YCB provides more reference value on cases that require a single model to handle multiple objects in a scene.

**Comment by imankgoyal** [comment]
Thank you so much for the information.
